x-pack/solutions/observability/plugins/observability_ai_assistant_app/scripts/evaluation/get_service_urls.ts

```diff
@@ -136,7 +136,7 @@ export async function getServiceUrls({
 
   const kibanaUrlWithAuth = format({
     ...parsedKibanaUrl,
-    auth: parsedKibanaUrl.auth || auth,
+    auth: parsedKibanaUrl.auth || 'elastic:changeme',
```

Comment:
What's the reason for this change? (`auth` => `elastic:changeme` as a fallback)

Reply:
Ah this was an issue I was facing when the `kibana.dev.yml` specifies elasticsearch credentials such as:
```
elasticsearch:
  hosts: <url>
  username: <username>
  password: <pw>
  ssl:
    verificationMode: none
server:
  host: 0.0.0.0
  restrictInternalApis: false
```
The credentials don't get picked up when it's specified like this. However, it gets picked up when specified as follows:
```
elasticsearch.hosts: http://localhost:9200
elasticsearch.username: <username>
elasticsearch.password: <pw>
elasticsearch.ssl.verificationMode: none
elasticsearch.ignoreVersionMismatch: true
```

We can discard this change.